controller-server/src/main/java/com/yahoo/vespa/hosted/controller/ApplicationController.java

```diff
@@ -293,9 +326,9 @@ public ActivateResult deploy(ApplicationId applicationId, ZoneId zone,
             Set<ContainerEndpoint> endpoints;
             Optional<ApplicationCertificate> applicationCertificate;
 
-            try (Lock lock = lock(TenantAndApplicationId.from(applicationId));
-                 Lock oldLock = lock(applicationId)) {
```

Comment:
The release before this one should make all instances lock on the application level (in addition to on the instance level), making it OK for this release to lock _only_ on application level.